core/Filesystem.php

```diff
@@ -20,12 +20,22 @@
  */
 class Filesystem
 {
+    /**
+     * @var bool
+     * @internal
+     */
+    public static $skipCacheClearOnUpdate = false;
+
     /**
      * Called on Core install, update, plugin enable/disable
      * Will clear all cache that could be affected by the change in configuration being made
      */
     public static function deleteAllCacheOnUpdate($pluginName = false)
     {
+        if (self::$skipCacheClearOnUpdate) {
+            return;
+        }
+
```

Comment:
Shouldn't we reload the plugin information nevertheless? Wondering if it might cause weird issues when a plugin is updated, but the information isn't reloaded.

Reply:
:+1: that makes sense

Reply:
skipping this change as discussed on slack